aspnet/tutorials/first-web-api.rst

```diff
@@ -155,32 +149,32 @@ These methods implement the two GET methods:
 
 Here is an example HTTP response for the ``GetAll`` method::
 
-    HTTP/1.1 200 OK
-    Content-Type: application/json; charset=utf-8
-    Server: Microsoft-IIS/10.0
-    Date: Thu, 18 Jun 2015 20:51:10 GMT
-    Content-Length: 82
+  HTTP/1.1 200 OK
+  Content-Type: application/json; charset=utf-8
+  Server: Microsoft-IIS/10.0
+  Date: Thu, 18 Jun 2015 20:51:10 GMT
+  Content-Length: 82
 
-    [{"Key":"4f67d7c5-a2a9-4aae-b030-16003dd829ae","Name":"Item1","IsComplete":false}]
+  [{"Key":"4f67d7c5-a2a9-4aae-b030-16003dd829ae","Name":"Item1","IsComplete":false}]
 
 Later in the tutorial I'll show how you can view the HTTP response using the Fiddler tool.
 
-Routing and URL paths 
+Routing and URL paths
 ^^^^^^^^^^^^^^^^^^^^^
 
 The `[HttpGet] <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/HttpGetAttribute/index.html>`_ attribute specifies that these are HTTP GET methods. The URL path for each method is constructed as follows:
 
 - Take the template string in the controller’s route attribute,  ``[Route("api/[controller]")]``
-- Replace "[Controller]" with the name of the controller, which is the controller class name minus the "Controller" suffix. For this sample the name of the controller is "todo" (case insensitive). For this sample, the controller class name is **Todo**\Controller and the root name is "todo". ASP.NET MVC is not case sensitive.
-- If the ``[HttpGet]`` attribute also has a template string, append that to the path. This sample doesn't use a template string. 
+- Replace "[Controller]" with the name of the controller, which is the controller class name minus the "Controller" suffix. For this sample the name of the controller is "todo" (case insensitive). For this sample, the controller class name is **Todo**\Controller and the root name is "todo". ASP.NET MVC Core is not case sensitive.
+- If the ``[HttpGet]`` attribute also has a template string, append that to the path. This sample doesn't use a template string.
 
 For the ``GetById`` method,  "{id}" is a placeholder variable. In the actual HTTP request, the client will use the ID of the ``todo`` item. At runtime, when MVC invokes ``GetById``, it assigns the value of "{id}" in the URL the method's ``id`` parameter.
 
 Open the *src\\TodoApi\\Properties\\launchSettings.json* file and replace the ``launchUrl`` value to use the ``todo`` controller. That change will cause IIS Express to call the ``todo`` controller when the project is started.
```

Comment:
You can update the URL that IIS Express launches at from within VS via the Debug tab of the project Properties. Let's show that instead of directly modifying launchSettings.json.